packages/block-library/src/categories/edit.js

```diff
@@ -92,9 +106,24 @@ export default function CategoriesEdit( {
 		const categoriesList = getCategoriesList( parentId );
 		return (
 			<>
-				<VisuallyHidden as="label" htmlFor={ selectId }>
-					{ __( 'Categories' ) }
-				</VisuallyHidden>
+				{ showLabel ? (
+					<RichText
+						className="wp-block-categories__label"
+						as="label"
```

Comment:
I'm not sure the `as` prop works for RichText. Instead, the RichText component has a `tagName` prop to change the element type. However, I'm not sure why we'd want to render this as a `label` element in the editor anyways, as the semantics is overridden by `role="textbox"`.

Reply:
The markup in the editor and front needs to match, otherwise it will be difficult to style the block.

Reply:
Ah, the recurring problem with using `<label>` in the editor is that clicking on it selects the input element.